source/connect/tls.txt

```diff
@@ -0,0 +1,272 @@
+  option to ``true``
+
+.. tabs::
+
+   .. tab:: Connection String
+      :tabid: connectionstring
+
+      .. literalinclude:: /includes/connect/tls.c
+         :language: c
+         :start-after: start-connect-str
+         :end-before: end-connect-str
+         :dedent:
+
+   .. tab:: MongoC URI Options
+      :tabid: mongocurioptions
+
+      .. literalinclude:: /includes/connect/tls.c
+         :language: c
+         :start-after: start-connect-uri-opt
+         :end-before: end-connect-uri-opt
+         :dedent:
+
+.. tip::
+   
+   If your connection string includes the ``+srv`` modification, which specifies the
+   SRV connection format, TLS is enabled on your connection by default.
+
+   To learn more about the SRV connection format, see
+   :manual:`SRV Connection Format </reference/connection-string/#srv-connection-format>`
+   in the {+mdb-server+} documentation.
+
+.. _c-specify-ca-file:
+
+Specify a CA File
+------------------
+
+When you connect to a MongoDB deployment with TLS enabled, the deployment will by default require the client to provide
+a client certificate issued by a certificate authority, or an authority
+trusted by the native certificate store in use on the server.
+
+You can provide the client certificate in the following ways:
+
+- Setting the ``tlscertificatekeyfile`` parameter in your connection string to a PEM armored certificate file
+- Using the `mongoc_uri_set_option_as_utf8()` function to set the `MONGOC_URI_TLSCERTIFICATEKEYFILE` option
+  to a `.pem` file containing the root certificate chain
+
+.. tabs::
+
+   .. tab:: Connection String
+      :tabid: connectionstring
+
+      .. literalinclude:: /includes/connect/tls.c
+         :language: c
+         :start-after: start-connect-ca-file-str
+         :end-before: end-connect-ca-file-str
+         :dedent:
+
+   .. tab:: MongoC URI Options
+      :tabid: mongocurioptions
+
+      .. literalinclude:: /includes/connect/tls.c
+         :language: c
+         :start-after: start-connect-ca-file-uri
+         :end-before: end-connect-ca-file-uri
+         :dedent:
+
+.. _c-certificate-revocation:
+
+Server Certificate Verification
+-------------------------------
+
+The MongoDB C Driver will automatically verify the validity of a server certificate issued
+by the configured Certificate Authority. The driver also performs hostname validation and revocation checking.
+
+To overwrite this behavior, it is possible to disable hostname validation, OCSP endpoint revocation checking,
+revocation checking entirely, and allow invalid certificates.
+
+This behavior is controlled using the ``tlsAllowInvalidHostnames``, ``tlsDisableOCSPEndpointCheck``,
+``tlsDisableCertificateRevocationCheck``, and ``tlsAllowInvalidCertificates`` options. By default,
+all are set to ``false``.
+
+It is not recommended to change these defaults as it exposes the client to *Man In The Middle* attacks (when
```

Comment:
S: I think this paragraph would be more readable as a list; you could also avoid parentheses that way. Something like: "It is not recommended to change these defaults, since you might expose your client to the following security risks:
- Man in the middle attacks, when ``tlsAllowInvalidHostnames`` is set
...